src/main/java/dk/sdu/se_f22/sortingmodule/scoring/Scoring.java

```diff
@@ -1,88 +1,278 @@
 package dk.sdu.se_f22.sortingmodule.scoring;
 
-import java.util.List;
+import java.sql.*;
+import java.util.*;
+import java.util.Date;
 
-public class Scoring implements IScoring{
+public class Scoring implements IScoring {
+    private static Scoring instance;
+    private String url = "localhost";
+    private int port = 5432;
+    private String databaseName = "testprojectdb";
+    private String username = "postgres";
+    private String password = "postgres582";
+    private Connection connection = null;
 
-    private void price(List<Object> input) {
-        for (Object o : input) {
-           /* int price = o.getPrice();
+    private Scoring(){
+        initializePostgresqlDatabase();
+    }
 
-            for (int j = 0; j < db.size(); j++) {
-                if (price < db(j).get(Price.bracket)) {
-                    o.setScore(o.getScore() + db(j).get(Price.weight));
-                }
-            }*/
+    public static Scoring getInstance(){
+        if (instance == null) {
+            instance = new Scoring();
+        }
+        return instance;
+    }
+
+    private void initializePostgresqlDatabase() {
+        try {
+            DriverManager.registerDriver(new org.postgresql.Driver());
+            connection = DriverManager.getConnection("jdbc:postgresql://" + url + ":" + port + "/" + databaseName, username, password);
```

Comment:
I would like you to use the common database connection, with is provided in the "DBConnection" -> `DBConnection.getPooledConnection();`
Then follow the readme in the main repository branch, where it says on how to configure this connection (through "config.properties"